### PR TITLE
Remove the header size when we calculate the scroll position to center highlighted elements.

### PR DESCRIPTION
**Pull request by yunabe**
This PR request centers highlighted nodes precisely even if the viewer header is shown.

### CODE REVIEW

File: extensions/amp-viewer-integration/0.1/highlight-handler.js [hunk 1]
```diff
@@ -212,15 +212,19 @@ export class HighlightHandler {
...
     let maxBottom = 0;
+    const paddingTop = viewport.getPaddingTop();
     for (let i = 0; i < nodes.length; i++) {
       const {top, bottom} = viewport.getLayoutRect(nodes[i]);
-      minTop = Math.min(minTop, top);
-      maxBottom = Math.max(maxBottom, bottom);
+      // top and bottom returned by getLayoutRect includes the header padding
+      // size. We need to cancel the padding to calculate the positions in
+      // document.body like Viewport.animateScrollIntoView does.
+      minTop = Math.min(minTop, top - paddingTop);
```
Comment (alanorozco) [inline extensions/amp-viewer-integration/0.1/highlight-handler.js]: Use `moveLayoutRectLtwh` for clarity

Comment (yunabe) [reply extensions/amp-viewer-integration/0.1/highlight-handler.js]: Thank you for the review. Rewrote with `moveLayoutRect`

**Comment by yunabe** [comment]
Could you merge this pull request? I don't have write access of this project.

**Comment by yunabe** [comment]
Thanks!

**Pull request merged**